### PR TITLE
Update from Scroll Ecosystem Data - 06/24/2024

### PR DESCRIPTION
**Pull request by dghelm**
Updates:
- various submissions and builds from hackathons
- several subecosystems for projects that have deployed to Scroll

### CODE REVIEW

File: data/ecosystems/s/scroll.toml [hunk 1]
```diff
@@ -484,21 +838,42 @@ url = "https://github.com/crackeryo/ethonline2023"
...
+url = "https://github.com/CryptoSnipper/scroll-dev-ai-bot"
+
+[[repo]]
+url = "https://github.com/cryptotwilight/3BD"
+
 [[repo]]
 url = "https://github.com/cryptotwilight/kuala-bond"
 
+[[repo]]
+url = "https://github.com/crystal-lang-tools/scry"
```
Comment (jubos) [inline data/ecosystems/s/scroll.toml:866]: This repo doesn't look applicable to scroll.

File: data/ecosystems/s/scroll.toml [hunk 2]
```diff
@@ -2339,39 +3518,69 @@ url = "https://github.com/thomaspanf/ethden23-Ls-with-frens"
...
+url = "https://github.com/ToJen/circuit-breaker"
+
 [[repo]]
 url = "https://github.com/TomKimKR/scaffold-eth_ETHTokyo"
 
 [[repo]]
 url = "https://github.com/tommymsz006/yubi25519"
 
+[[repo]]
+url = "https://github.com/tomochain/tomochain"
```
Comment (jubos) [inline data/ecosystems/s/scroll.toml:3561]: Tomochain doesn't look applicable either.

File: data/ecosystems/s/scroll.toml [hunk 3]
```diff
@@ -2409,6 +3627,12 @@ url = "https://github.com/unifralabs/scroll-contract-verify-demo"
 [[repo]]
 url = "https://github.com/unifralabs/scroll-explorer-l2-indexer"
 
+[[repo]]
+url = "https://github.com/unlock-protocol/unlock"
```
Comment (jubos) [inline data/ecosystems/s/scroll.toml:3631]: This can be added as a sub ecosystem.

**Review by jubos — CHANGES_REQUESTED** [review]
Looking pretty good -- just has a few repos that should be removed.

File: data/ecosystems/s/scroll.toml (hunk 1 above)
Comment (dghelm) [reply data/ecosystems/s/scroll.toml]: Removed.

File: data/ecosystems/s/scroll.toml (hunk 2 above)
Comment (dghelm) [reply data/ecosystems/s/scroll.toml]: Removed.

File: data/ecosystems/s/scroll.toml (hunk 3 above)
Comment (dghelm) [reply data/ecosystems/s/scroll.toml]: Removed and added to Subecosystem. Good catch.

**Comment by dghelm** [comment]
@jubos please let me know if there's anything else you need from our end!

**Pull request merged**